.github/actions/publish/action.yml

```
@@ -18,12 +18,13 @@ runs:
   using: composite
   steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Setup Node.js
       uses: actions/setup-node@v3
       with:
         node-version: '16'
```

依存関係のキャッシュをする場合は明示的な指定が必要なので指定するようにしてください(他も同様)
ref: https://github.com/actions/setup-node#caching-global-packages-data
```suggestion
        node-version: '16'
        cache: yarn
```

---

[`d839db6` (#651)](https://github.com/approvers/OreOreBot2/pull/651/commits/d839db6f6fc89a019201edfbbde7b186af81c599) でまとめて修正しました